webxr/openxr/input.rs

```diff
@@ -190,32 +194,35 @@ impl OpenXRInput {
             needs_hands,
         );
 
-        let mut bindings =
-            right_hand.get_bindings(instance, "trigger/value", Some("squeeze/click"));
-        bindings.extend(
-            left_hand
-                .get_bindings(instance, "trigger/value", Some("squeeze/click"))
-                .into_iter(),
-        );
-        let path_controller = instance
-            .string_to_path("/interaction_profiles/microsoft/motion_controller")
-            .unwrap();
-        instance
-            .suggest_interaction_profile_bindings(path_controller, &bindings)
-            .unwrap();
+        INTERACTION_PROFILES.iter().for_each(|profile| {
+            if let Some(extension_name) = profile.required_extension {
+                if !supported_interaction_profiles
+                    .contains(&String::from_utf8(extension_name.to_vec()).unwrap())
```

Comment:
I think it should be possible to use str::from_utf8 and avoid the to_vec and String allocations.